src/mixs/schema/mixs.yaml

```diff
@@ -14737,7 +14755,288 @@ slots:
       syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
       interpolated: true
       partial_match: true
+  isotope:
+    title: isotope
+    description: The target isotope for measurement in the experimental sample. List both the element and the atomic mass.
+    range: IsotopeEnum
+    multivalued: true
+    examples:
+      - value: 13C
+      - value: 13C; 15N
```

Comment:
slot_uri: MIXS:###

These are missing from all my new slots. Do you know how these are made / assigned? Just sequential?